src/pipeline.mjs

```diff
@@ -1,3 +1,3 @@
-export default function pipeline (...fns) {
-  return fns.reverse().reduce((f, g) => (...args) => f(g(...args)))
+export default function pipeline (iterable, ...fns) {
```

Comment:
About the argument name "iterable": While users of `iter-tools` mostly focus on iterables, this function is applicable to other types as well, the following code is valid for example:

```javascript
const result = pipeline(
  3,
  x => x * 2,
  x => x + 1,
  x => x * x
)
```